Doc/library/threading.rst

```diff
@@ -16,9 +16,9 @@ level :mod:`_thread` module.  See also the :mod:`queue` module.
 
 .. note::
 
-   While they are not listed below, the ``camelCase`` names used for some
+   The ``camelCase`` names used for some
    methods and functions in this module in the Python 2.x series are still
-   supported by this module.
+   supported by this module. They will be removed in the future.
```

Comment:
Please don't schedule removal and explain that these aliases kept for backward compatibility with Python 2 are deprecated since Python 3.10.

Reply:
Sure. Just to be clear though, this would be compatibility with Python *2.5* and lower: the non-camel-case versions have existed since 2.6 (https://github.com/python/cpython/blob/2.6/Lib/threading.py).

Also, thanks for your review! I'll make the requested changes later today.

Reply:
Please clarify that the aliases are kept for backward compatibility with Python 2.5 in this case ;-)